cpp/tests/utilities/column_wrapper.hpp

```diff
@@ -423,13 +423,55 @@ class fixed_width_column_wrapper : public detail::column_wrapper {
     : fixed_width_column_wrapper(begin, end, std::cbegin(validity))
   {
   }
+
+  /**
+   * @brief Construct a nullable column from a list of pairs of fixed-width
+   * elements and validity booleans of each element.
+   *
+   * The validity of each element is determined by the boolean element in the pair
+   * where `true` indicates the element is valid, and `false` indicates the
+   * element is null.
+   *
+   * Example:
+   * @code{.cpp}
+   * // Creates a nullable INT32 column with 4 elements: {1, NULL, 3, NULL}
+   * using p = std::pair<int32_t, bool>;
+   * fixed_width_column_wrapper<int32_t> w( p{1, true}, p{2, false}, p{3, true}, p{4, false} );
+   * @endcode
+   *
+   * @param elements The list of pairs of element and validity booleans
+   */
+  template <typename ElementFrom>
+  fixed_width_column_wrapper(std::initializer_list<std::pair<ElementFrom, bool>> elements)
+  {
+    auto begin =
+      thrust::make_transform_iterator(elements.begin(), [](auto const& e) { return e.first; });
+    auto end = begin + elements.size();
+    auto v =
+      thrust::make_transform_iterator(elements.begin(), [](auto const& e) { return e.second; });
+    wrapped = fixed_width_column_wrapper<ElementTo, ElementFrom>(begin, end, v).release();
+  }
 };
 
 /**
  * @brief `column_wrapper` derived class for wrapping columns of strings.
  **/
 class strings_column_wrapper : public detail::column_wrapper {
  public:
+  /**
+   * @brief Default constructor initializes an empty column of strings
+   **/
+  strings_column_wrapper() : column_wrapper{}
+  {
+    std::vector<std::string> empty;
+    std::vector<char> chars;
+    std::vector<int32_t> offsets;
+    auto all_valid = make_counting_transform_iterator(0, [](auto i) { return true; });
+    std::tie(chars, offsets) =
+      detail::make_chars_and_offsets(empty.begin(), empty.end(), all_valid);
+    wrapped = cudf::make_strings_column(chars, offsets);
+  }
+
```

Comment:
Forward to the init list ctor and make the init list type explicit to avoid ambiguity. 
```suggestion
  strings_column_wrapper() : strings_column_wrapper( std::initializer_list<std::string>{} ) {}
```